pkgs/development/python-modules/moderngl_window/default.nix

```diff
@@ -0,0 +1,42 @@
+{ lib
+, buildPythonPackage
+, fetchFromGitHub
+, isPy3k
+, numpy
+, moderngl
+, pyglet
+, pillow
+, pyrr
+, pytest
+}:
+
+buildPythonPackage rec {
+  pname = "moderngl_window";
+  version = "1.2.0";
+
+  src = fetchFromGitHub {
+    owner = "moderngl";
+    repo = pname;
+    rev = version;
+    sha256 = "054w77lyc2nc0dyx76zsrbq2b3xbywdijhb62b2qqm99ldr1k1x5";
+  };
+
+  propagatedBuildInputs = [ numpy moderngl pyglet pillow pyrr ];
+
+  disabled = !isPy3k;
+
+  # Tests need a display to run.
+  doCheck = false;
+  checkInputs = [ pytest ];
```

Comment:
Any reason to include ``checkInputs`` when you disable the tests?

Reply:
Included it because if in the future it becomes possible to run these tests (without a X server), it is known how to run these tests.

Reply:
I think we can remove it and save some space and time during the builds. In the future whoever decides to implement these tests can add it.

Reply:
Ok, removed it.